### PR TITLE
vikyaiyer.json added for registering 'is a dev' domain

### PR DESCRIPTION
**Pull request by vikyaiyer**
Thanks for the free domain

### CODE REVIEW

File: domains/vikyaiyer.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "description": "Vignesh Aiyer is a Developer!",
+    "repo": "https://github.com/vikyaiyer/jekyll-theme-minimal-resume",
+    "owner": {
+      "username": "vikyaiyer",
+      "email": "vikyaiyer1997@gmail.com"
+    },
+    "record": {
+      "CNAME": "https://vikyaiyer.github.io/jekyll-theme-minimal-resume",
```
Comment (phenax) [inline domains/vikyaiyer.json:9]: CNAME records cannot be a full url. It has to be a host name. I'd suggest you rename your repo to `vikyaiyer.github.io`.

You can take a look at the [github pages guide](https://guides.github.com/features/pages/).

**Review by phenax — CHANGES_REQUESTED** [review]

File: domains/vikyaiyer.json (hunk 1 above)
Comment (vikyaiyer) [reply domains/vikyaiyer.json]: I have updated the url

Comment (phenax) [reply domains/vikyaiyer.json]: Remove the `https://` from the cname and it'll be good to merge

Comment (vikyaiyer) [reply domains/vikyaiyer.json]: removed✌️

**Review by phenax — APPROVED** [review]

**Pull request merged**